.builder/actions/crt_size_check.py

```diff
@@ -11,7 +11,7 @@ def run(self, env):
         # Maximum package size (for current platform) in bytes
         # NOTE: if you increase this, you might also need to increase the
         # limit in continuous-delivery/pack.sh
-        max_size = 7_500_000
+        max_size = 8_000_000
```

Comment:
Do we need to increase the limit in pack.sh?

Reply:
don't know... the pack.sh is not exactly match the CI here, you could try to run the test pipeline see if we need to

Reply:
We will find out

Reply:
We found out in #587